Source/EasyNetQ/PublishConfiguration.cs

```diff
@@ -12,22 +14,29 @@ public interface IPublishConfiguration
         /// Sets a priority of the message
         /// </summary>
         /// <param name="priority">The priority to set</param>
-        /// <returns>IPublishConfiguration</returns>
+        /// <returns>Returns a reference to itself</returns>
         IPublishConfiguration WithPriority(byte priority);
 
         /// <summary>
         /// Sets a topic for the message
         /// </summary>
         /// <param name="topic">The topic to set</param>
-        /// <returns>IPublishConfiguration</returns>
+        /// <returns>Returns a reference to itself</returns>
         IPublishConfiguration WithTopic(string topic);
 
         /// <summary>
         /// Sets a TTL for the message
         /// </summary>
         /// <param name="expires">The TTL to set in milliseconds</param>
-        /// <returns>IPublishConfiguration</returns>
+        /// <returns>Returns a reference to itself</returns>
         IPublishConfiguration WithExpires(int expires);
+
+        /// <summary>
+        /// Sets request headers
```

Comment:
Note that this copy-pasted comment from `IRequestConfiguration` may be misleading for other configurations.